satellite/orders/service.go

```diff
@@ -161,6 +161,12 @@ func (service *Service) CreateGetOrderLimits(ctx context.Context, uplink *identi
 			continue
 		}
 
+		if node.Disqualified != nil {
+			service.log.Debug("node is disqualified", zap.Error(err))
```

Comment:
There is no `err` returned when checking the disqualified field. Log the Node ID instead -- see the IsOnline check for reference. Same for combining the errors.